Shared/Components/LetterPickerOrientation.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import SwiftUI
+
+enum LetterPickerOrientation: String, CaseIterable, Defaults.Serializable, Displayable {
+
+    case none
```

Comment:
Instead of `none`, for this I'd like there to be a `LetterPickerEnabled` bool option and this is just the orientation.

I would like the default to be `false`.

Reply:
How should I lay this out? Does this look like what you were expecting or should this be more of a cascading menu like the Library/Search Filters below it? Disregard the duplicate labels, this would be Letter Picker and Orientation respectively.

![simulator_screenshot_A3D1648F-DF3D-423E-9C8A-463C39DC8140](https://github.com/jellyfin/Swiftfin/assets/37599297/0e27efac-0bab-400b-8544-ee254d8dbd66)